GTG/tools/autostart.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Getting Things GNOME! - a personal organizer for the GNOME desktop
+# Copyright (c) 2008-2013 - Lionel Dricot & Bertrand Rousseau
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program.  If not, see <http://www.gnu.org/licenses/>.
+# -----------------------------------------------------------------------------
+
+import os
+import shutil
+
+from xdg.BaseDirectory import xdg_config_home
+
+AUTOSTART_DIRECTORY = os.path.join(xdg_config_home, "autostart")
+AUTOSTART_FILE = "gtg.desktop"
+AUTOSTART_PATH = os.path.join(AUTOSTART_DIRECTORY, AUTOSTART_FILE)
+
+
+class Autostart(object):
+    def enable_gtg_autostart():
+        """ Enable autostart
+
+        Firstly, locate gtg.desktop file. Then link it in AUTOSTART_FILE.
+        On Windows, there is no os.symlink, just copy the file. """
+        desktop_file_path = None
+        this_directory = os.path.dirname(os.path.abspath(__file__))
+        for path in ["../..", "../../../applications",
```

Comment:
Now that you now how to do unit tests, do you mind to write unit tests for this module? It is pretty separate so it should be simple to write unit tests. (With lot of mockups for os operations).